src/test/java/io/stargate/sgv2/jsonapi/service/operation/collections/FindCollectionOperationTest.java

```diff
@@ -3018,15 +3018,15 @@ public void readFailureException() throws UnknownHostException {
           ThrowableToErrorMapper.getMapperWithMessageFunction()
               .apply(failure, failure.getMessage());
       AssertionsForClassTypes.assertThat(error).isNotNull();
-      AssertionsForClassTypes.assertThat(error.message())
```

Comment:
Just reordered to start with more general tests (error code, exception class). And then changed testing for message according to change in `ThrowableToErrorMapper`